airbyte-cdk/bulk/toolkits/load-iceberg-parquet/src/main/kotlin/io/airbyte/cdk/load/data/iceberg/parquet/AirbyteValueToIcebergRecord.kt

```diff
@@ -20,6 +20,9 @@ import org.apache.iceberg.data.GenericRecord
 import org.apache.iceberg.types.Type
 
 class AirbyteValueToIcebergRecord {
+
+    private val timeStringUtility = TimeStringUtility()
```

Comment:
If we are not going to let Micronaut manage/inject this, it may make sense to make it a Kotlin object instead of a class to make it a singleton.  That would allow us to reference the methods in a static-like way.